pspmutating.go

```diff
@@ -0,0 +1,133 @@
+package pspmigrator
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/go-test/deep"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/api/policy/v1beta1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+)
+
+func GetContainerSecurityContexts(podSpec v1.PodSpec) []*v1.SecurityContext {
+	scs := make([]*v1.SecurityContext, 0)
+	for _, c := range podSpec.Containers {
```

Comment:
This should probably include `InitContainers` and maybe `EphemeralContainers`. Alternatively, consider importing and reusing `VisitContainers` https://github.com/kubernetes/kubernetes/blob/master/pkg/api/pod/util.go#L64

Reply:
done and added TODO for re-using VisitContainers since that might not be as straightforward